prime-dotnet-webapi/Configuration/AccessTerms/LicenseClassClauseConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using Prime.Models;
+
+namespace Prime.Configuration
+{
+    public class LicenseClassClauseConfiguration : SeededTable<LicenseClassClause>
+    {
+        public override ICollection<LicenseClassClause> SeedData
+        {
+            get
+            {
+                return new[] {
+                    new LicenseClassClause { Id = 1, Clause = "", Type = "Dispense", EffectiveDate = SEEDING_DATE, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
```

Comment:
Type should be a constant so it can be searched on.

Reply:
I'm thinking on LicenceClassClause rather than in PrimeConstants? I'm not 100% sure